rest_docs/rest-catalog-open-api.yaml

```diff
@@ -0,0 +1,849 @@
+            example: {
+              "error": {
+                "message": "The server does not support this operation",
+                "type": "UnsupportedOperationException",
+                "code": 406
+              } }
+
+    DropTableResponse:
+      description: A successful call to drop a table
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              dropped:
+                type: boolean
+                description: true if the table was found and removed from the catalog
+              purged:
+                type: boolean
+                description: whether the underlying data was purged or is being purged
+
+    DropNamespaceResponse:
+      description: A successful call to drop a namespace
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              dropped:
+                description: true if the namespace was dropped
+                type: boolean
+
+    GetNamespaceResponse:
+      description:
+        Returns a namespace, as well as any properties stored on the namespace if namespace properties
+        are supported by the server.
+      content:
+        application/json:
+          schema:
+            type: object
+            required:
+              - namespace
+            properties:
+              namespace:
+                $ref: '#/components/schemas/Namespace'
+              properties:
+                type: object
+                description:
+                  Properties stored on the namespace, if supported by the server.
+                  If the server does not support namespace properties, it should return null for this field.
+                  If namespace properties are supported, but none are set, it should return an empty object.
+                example: { "owner": "Ralph", 'transient_lastDdlTime': '1452120468' }
+                default: { }
+                nullable: true
+
+    ListTablesResponse:
+      description: A list of table identifiers
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              identifiers:
+                type: array
+                uniqueItems: true
+                items:
+                  $ref: '#/components/schemas/TableIdentifier'
+
+    ListNamespacesResponse:
+      description: A list of namespaces
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              namespaces:
+                type: array
+                uniqueItems: true
+                items:
+                  $ref: '#/components/schemas/Namespace'
+
+    UpdateNamespacePropertiesResponse:
+      description: JSON data response for a synchronous update properties request.
+      content:
+        application/json:
+          schema:
+            type: object
+            required:
+              - updated
```

Comment:
I think these should not be required, because you might have only updates or removes.

Reply:
I think this comment applies here as well: https://github.com/apache/iceberg/pull/3770/files#r777620121

I'd probably leave these as required, but it is minor to me.